robots/commenter/main.go

```diff
@@ -242,29 +245,51 @@ func run(c client, query, sort string, asc, random bool, commenter func(meta) (s
 			break
 		}
 		log.Printf("Matched %s (%s)", i.HTMLURL, i.Title)
-		org, repo, number, err := parseHTMLURL(i.HTMLURL)
-		if err != nil {
-			msg := fmt.Sprintf("Failed to parse %s: %v", i.HTMLURL, err)
-			log.Print(msg)
-			problems = append(problems, msg)
-		}
-		comment, err := commenter(meta{Number: number, Org: org, Repo: repo, Issue: i})
-		if err != nil {
-			msg := fmt.Sprintf("Failed to create comment for %s/%s#%d: %v", org, repo, number, err)
-			log.Print(msg)
-			problems = append(problems, msg)
-			continue
+		if err := createComment(c, commenter, spamCeiling, i); err != nil {
```

Comment:
I think we'd like the issues/PRs skipped like this to not being counted into the `--ceiling` limit

Reply:
This isn't related to this PR as the ceiling was always kept the same (`n` was being incremented for every interval regardless of a success or a failure). Anyway, I'll fix that.

Reply:
I asked for this because  previously the loop did not do any decisions about whether it wants to comment on the PRs or not: all cases when it didn't were errors.